lib/cache.js

```diff
@@ -69,10 +69,13 @@ module.exports = class Cache {
       )
     }
 
-    return content.replace(
+    content = content.replace(
       matches[0],
-      `${this.config.url}/download/nupkg?update=true`
+      `${this.config.url}/download/latest/${matches[0]}`
```

Comment:
change
`${this.config.url}/download/latest/${matches[0]}`
to
`https://${this.config.url}/download/latest/${matches[0]}`

To avoid/fix Squirrel 4294967295 error: Filename must be either an abosolute URL or a file name

Reply:
Can confirm that PR works. Any plans on merging it to master?